sdv/single_table/base.py

```diff
@@ -269,6 +270,26 @@ def get_metadata(self):
         """Return the ``SingleTableMetadata`` for this synthesizer."""
         return self.metadata
 
+    def add_constraints(self, constraints):
+        """Add constraints to the synthesizer.
+
+        Args:
+            constraints (list):
+                List of constraints described as dictionaries in the following format:
+                    * ``constraint_class``: Name of the constraint to apply.
+                    * ``constraint_parameters``: A dictionary with the constraint parameters.
+        """
+        if self._fitted:
+            warnings.warn(
+                "For these constraints to take effect, please refit the synthesizer using 'fit'."
+            )
+
+        self._data_processor.add_constraints(constraints)
+
+    def get_constraints(self):
+        """Return a list of the current constraints that will be used."""
```

Comment:
Add `Returns:` section, since it's a public method.

Reply:
👍🏻